### PR TITLE
added login and registraction form

### PR DESCRIPTION
**Pull request by mfukechristian**
## Naming Rules

Name your PR like this: LOGIN & REGISTRATION FORMS -CHRISTIAN MFUKE

## Description

I have created a skeleton of the login and  registration form to help the user   log in and  subscribe 

## Related to

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

Fixes #1

## Checklist:

- [* ] My code follows the style guidelines of this project
- [ *] I have carefully reviewed my own code
- [ *] I have commented my code
- [ *] I have updated any documentation

### CODE REVIEW

File: client/src/pages/About.js [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
 			</p>
-			<a href="https://github.com/textbook/starter-kit/wiki">Wiki</a>
+			
+			<Link to="/login/this/site">
+				<Button variant="primary" style={{ width:"150px",margin:"20px",backgroundColor:"#0F172A",border:"none",height:"50px"}}>Log In</Button>
+			</Link>
+
+			<Link to="/registraction/this/site">
+				<Button variant="primary" style={{ width:"150px",margin:"20px",backgroundColor:"#0F172A",border:"none",height:"50px" }}>Registraction</Button>
+			</Link>
```
Comment (Craques) [inline client/src/pages/About.js:18]: Looks good, maybe we may tie this to the navigation bar, so this change may not be necessary

File: client/src/pages/Registraction.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import Button from "react-bootstrap/Button";
+import Form from "react-bootstrap/Form";
+
+const Registraction = () => {
+  return (
+    <> <h1 style={{ textAlign:"center",marginTop:"50px" }}>Registraction Form</h1>
+    <Form style={{ width:"500px",margin:"100px auto" }}>
```
Comment (Craques) [inline client/src/pages/Registraction.js:8]: This looks good, can we maybe not used fixed widths, you can use the `Column` component and use some of the available breakpoints e.g `md`, `sm`

File: client/src/App.js [hunk 3]
```diff
@@ -2,11 +2,15 @@ import { Route, Routes } from "react-router-dom";
...
 import Home from "./pages/Home";
+import Login from "./pages/Login";
+import Registraction from "./pages/Registraction";
 
 const App = () => (
 	<Routes>
 		<Route path="/" element={<Home />} />
 		<Route path="/about/this/site" element={<About />} />
+		<Route path="/login/this/site" element={<Login />} />
+		<Route path="/registraction/this/site" element={<Registraction/>} />
```
Comment (Craques) [inline client/src/App.js:13]: nitpick, maybe we can change `registraction` to registration

**Review by Craques — APPROVED** [review]

**Review by ChrisKoetzee — APPROVED** [review]

**Review by ChrisKoetzee — APPROVED** [review]

**Review by ChrisKoetzee — APPROVED** [review]

**Review by ChrisKoetzee — DISMISSED** [review]

**Review by ChrisKoetzee — APPROVED** [review]

File: client/src/pages/About.js (hunk 1 above)
Comment (mfukechristian) [reply client/src/pages/About.js:18]: I have fixed it please check my pull request

File: client/src/App.js [hunk 4]
```diff
@@ -3,14 +3,14 @@ import { Route, Routes } from "react-router-dom";
...
-import Registraction from "./pages/Registraction";
+import Registration from "./pages/Registration";
 
 const App = () => (
 	<Routes>
 		<Route path="/" element={<Home />} />
 		<Route path="/about/this/site" element={<About />} />
 		<Route path="/login/this/site" element={<Login />} />
-		<Route path="/registraction/this/site" element={<Registraction/>} />
+		<Route path="/registraction/this/site" element={<Registration/>} />
```
Comment (Craques) [inline client/src/App.js:13]: maybe add the path as /student-registration

**Review by Craques — DISMISSED** [review]

File: client/src/App.js (hunk 4 above)
Comment (Craques) [reply client/src/App.js:13]: otherwise looks good

**Review by ChrisKoetzee — APPROVED** [review]

**Review by ChrisKoetzee — APPROVED** [review]

File: client/src/App.js (hunk 4 above)
Comment (ChrisKoetzee) [reply client/src/App.js]: Hi @mfukechristian , not sure if you are aware, but we have approved your changes, but you have to resolve the conversations, fix the es lint as well as the branch conflicts and then push again as we need to review again

File: server/student.js [hunk 5]
```diff
@@ -71,5 +71,23 @@ router.put('/:id', (req, res) => {
...
+		return res.status(400).json({ error: 'An error occurred' });
+	  }
+	  if (results.rowCount === 0) {
+		return res.status(401).json({ error: 'Invalid email or password' });
+	  }
+	  const student = results.rows[0];
+	  const token = jwt.sign({ id: student.id }, process.env.JWT_SECRET);
+	  return res.status(200).json({ token });
+	});
+  });
```
Comment (Craques) [inline server/student.js:89]: This looks amazing

File: client/src/App.js (hunk 4 above)
Comment (mfukechristian) [reply client/src/App.js]: yea I saw it I have been struggling to  merge  can we do it tomorrow please

**Pull request closed without merging**